mindsdb/migrations/versions/2024-07-19_45eb2eb61f70_add_provider_to_agent.py

```diff
@@ -0,0 +1,55 @@
+"""add provider to agent
+
+Revision ID: 45eb2eb61f70
+Revises: 459a4cd24933
+Create Date: 2024-07-19 00:48:47.629700
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import table, select, update
+import mindsdb.interfaces.storage.db  # noqa
+
+
+# revision identifiers, used by Alembic.
+revision = '45eb2eb61f70'
+down_revision = '459a4cd24933'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('agents', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('provider', sa.String(), nullable=True))
+
+    with op.batch_alter_table('chat_bots', schema=None) as batch_op:
```

Comment:
db.py doesn't have changes about chat_bots table

Reply:
This is from auto generated changes, let me remove